src/app/pages/properties/properties.component.ts

```diff
@@ -0,0 +1,47 @@
+import { PropertiesService } from './../../shared/services/properties/properties.service';
+import { Component, OnInit } from "@angular/core";
+import { Property } from "../../models/Property";
+
+@Component({
+  selector: "app-properties",
+  templateUrl: "./properties.component.html",
+  styleUrls: ["./properties.component.scss"]
+})
+export class PropertiesComponent implements OnInit {
+  properties: Property[];
+  propertiesUrl = "http://127.0.0.1:8000/api/v1/properties/";
+  next: string;
+  previous: string;
+  toggle: boolean = true;
+
+  constructor(private propertiesServices: PropertiesService) {}
+
```

Comment:
I thought the team agreed to use Reactive Forms instead of Template Driven Forms

Reply:
Hey @kelvinndmo , could you explain a bit more on this because I am not using any forms in this feature.

Reply:
I didn't realize that. Thanks for the clarification